pages/treeherder.py

```diff
@@ -115,6 +119,9 @@ class ResultSet(Region):
         _runnable_jobs_locator = (By.CSS_SELECTOR, '.runnable-job-btn.filter-shown')
         _set_bottom_of_range_locator = (By.CSS_SELECTOR, '.open ul > li:nth-child(8) > a')
         _set_top_of_range_locator = (By.CSS_SELECTOR, '.open ul > li:nth-child(7) > a')
+        _test_busted_job_title_locator = (By.CSS_SELECTOR, 'span.group-job-list .btn-purple')
```

Comment:
I'm just wondering if we should use this CSS selector `span.group-job-list .btn-purple`. 
Because sometimes if you have no name like 'Mochitest', 'TestCluster', 'Reftest' in front of Failure color button there is `td` tag, instead of `span`.

Plus `.btn-purple` is for `exception` failures :)